docs/setup.md

```diff
@@ -1,92 +1,156 @@
-[Home](../README.md) |
+|[Home](../README.md) |
 |--------------------------------------------|
 
 # Installation
 
 1. To install a solution pack, click **Content Hub** > **Discover**.
-2. From the list of solution pack that appears, search for and select **Outbreak Response Framework**.
+2. From the list of solution packs that appear, search **Outbreak Response Framework**.
 3. Click the **Outbreak Response Framework** solution pack card.
 4. Click **Install** on the lower part of the screen to begin the installation.
 
 ## Prerequisites
-The **Outbreak Response Framework** solution pack depends on the following solution packs that are installed automatically &ndash; if not already installed.
+The **Outbreak Response Framework** solution pack depends on the following solution packs. These solution packs are installed automatically &ndash; if not already installed.
 
-| Name           | Type          | Version | Purpose                                          |
-|:---------------|:--------------|:--------|:-------------------------------------------------|
-| SOAR Framework | Solution Pack | v2.2.0  | Required for Incident Response modules           |
-| Threat Intel Management         | Solution Pack     | v1.1.0  | Required to ingest threat feeds                   |
-| Vulnerability Management     | Solution Pack     | v1.2.1  | Required to ingest CVE's for KEV's |
+| Name                     | Type          | Version | Purpose                                |
+|:-------------------------|:--------------|:--------|:---------------------------------------|
+| SOAR Framework           | Solution Pack | v2.2.0  | Required for Incident Response modules |
```

Comment:
SFSP version can be removed. We support earlier version as well. 
https://mantis.fortinet.com/bug_view_page.php?bug_id=0961045